src/main/java/com/synopsys/integration/alert/channel/jira/server/JiraServerIssueConfigValidator.java

```diff
@@ -0,0 +1,161 @@
+            try {
+                List<ProjectComponent> projectsResponseModel = projectService.getProjectsByName(jiraProjectName);
+                Optional<ProjectComponent> optionalProject = projectsResponseModel
+                                                                 .stream()
+                                                                 .filter(project -> jiraProjectName.equals(project.getName()) || jiraProjectName.equals(project.getKey()))
+                                                                 .findAny();
+                if (optionalProject.isPresent()) {
+                    return optionalProject.get();
+                } else {
+                    fieldErrors.put(JiraServerDescriptor.KEY_JIRA_PROJECT_NAME, String.format("No project named '%s' was found", jiraProjectName));
+                }
+            } catch (IntegrationException e) {
+                fieldErrors.put(JiraServerDescriptor.KEY_JIRA_PROJECT_NAME, String.format(CONNECTION_ERROR_FORMAT_STRING, "projects"));
+            }
+        } else {
+            requireField(fieldErrors, JiraServerDescriptor.KEY_JIRA_PROJECT_NAME);
+        }
+        return null;
+    }
+
+    private String validateIssueCreator(FieldAccessor fieldAccessor, Map<String, String> fieldErrors) {
+        Optional<String> optionalIssueCreator = fieldAccessor.getString(JiraServerDescriptor.KEY_ISSUE_CREATOR)
+                                                    .filter(StringUtils::isNotBlank)
+                                                    .or(() -> fieldAccessor.getString(JiraServerDescriptor.KEY_SERVER_USERNAME));
+        if (optionalIssueCreator.isPresent()) {
+            String issueCreator = optionalIssueCreator.get();
+            try {
+                boolean isValidJiraEmail = userSearchService.findUserByUsername(issueCreator)
+                                               .stream()
+                                               .map(UserDetailsResponseModel::getName)
+                                               .anyMatch(emailAddress -> emailAddress.equals(issueCreator));
+                if (isValidJiraEmail) {
+                    return issueCreator;
+                } else {
+                    fieldErrors.put(JiraServerDescriptor.KEY_ISSUE_CREATOR, String.format("The username '%s' is not associated with any valid Jira users.", issueCreator));
+                }
+            } catch (IntegrationException e) {
+                fieldErrors.put(JiraServerDescriptor.KEY_ISSUE_CREATOR, String.format(CONNECTION_ERROR_FORMAT_STRING, "users"));
+            }
+        } else {
+            requireField(fieldErrors, JiraServerDescriptor.KEY_ISSUE_CREATOR);
+        }
+        return null;
+    }
+
+    private String validateIssueType(FieldAccessor fieldAccessor, Map<String, String> fieldErrors) {
+        String issueType = fieldAccessor.getString(JiraServerDescriptor.KEY_ISSUE_TYPE).orElse(JiraServerDistributionUIConfig.DEFAULT_ISSUE_TYPE);
```

Comment:
I'm wondering if we should ensure that the project has this IssueType assigned to it as well. E.g. A custom issue type may not be available on all projects.